apps/commudle-admin/src/app/feature-modules/users/components/public-profile/public-profile.component.scss

```diff
@@ -11,6 +11,9 @@
 
       nb-icon {
         @apply md:com-text-xl;
+        &.status-default {
+          color: #595867;
+        }
```

Comment:
why we are using this?

Reply:
Because the default icon color is light and as suggested by shivendra the color of inactive icons was supposed to be dark